Packs/ReversingLabs_Titanium_Cloud/Integrations/ReversingLabsTitaniumCloudv2/ReversingLabsTitaniumCloudv2.py

```diff
@@ -1521,6 +1522,717 @@ def url_analyses_feed_from_date_output(response, start_time):
+        indicator=domain,
+        indicator_type=DBotScoreType.DOMAIN,
+        integration_name="ReversingLabs TitaniumCloud v2",
+        score=0,
+        reliability=RELIABILITY
+    )
+
+    indicator = Common.Domain(
+        domain=domain,
+        dbot_score=dbot_score
+    )
+
+    results = CommandResults(
+        outputs_prefix="ReversingLabs",
+        outputs={"domain_downloaded_files": response},
+        readable_output=markdown,
+        indicator=indicator
+    )
+
+    return results
+
+
+def domain_urls_command():
+    domain_ti = create_domain_ti_object()
+
+    domain = demisto.getArg("domain")
+    limit = int(demisto.getArg("result_limit"))
+    per_page = int(demisto.getArg("results_per_page"))
+
+    try:
+        response = domain_ti.urls_from_domain_aggregated(
+            domain=domain,
+            results_per_page=per_page,
+            max_results=limit
+        )
+    except Exception as e:
+        return_error(str(e))
+
+    results = domain_urls_output(response=response, domain=domain)
+    return_results(results)
+
+
+def domain_urls_output(response, domain):
+    urls_table = tableToMarkdown(
+        name="URL list",
+        t=response
+    )
+
+    markdown = f"## ReversingLabs URL-s associated with domain {domain}\n {urls_table}"
+
+    dbot_score = Common.DBotScore(
+        indicator=domain,
+        indicator_type=DBotScoreType.DOMAIN,
+        integration_name="ReversingLabs TitaniumCloud v2",
+        score=0,
+        reliability=RELIABILITY
+    )
+
+    indicator = Common.Domain(
+        domain=domain,
+        dbot_score=dbot_score
+    )
+
+    results = CommandResults(
+        outputs_prefix="ReversingLabs",
+        outputs={"domain_urls": response},
+        readable_output=markdown,
+        indicator=indicator
+    )
+
+    return results
+
+
+def domain_to_ip_command():
+    domain_ti = create_domain_ti_object()
+
+    domain = demisto.getArg("domain")
+    limit = int(demisto.getArg("result_limit"))
+    per_page = int(demisto.getArg("results_per_page"))
+
+    try:
+        response = domain_ti.domain_to_ip_resolutions_aggregated(
+            domain=domain,
+            results_per_page=per_page,
+            max_results=limit
+        )
+    except Exception as e:
+        return_error(str(e))
+
+    results = domain_to_ip_output(response=response, domain=domain)
+    return_results(results)
+
+
+def domain_to_ip_output(response, domain):
+    ip_table = tableToMarkdown(
+        name="IP address list",
+        t=response
+    )
+
+    markdown = f"## ReversingLabs IP addresses resolved from domain {domain}\n {ip_table}"
+
+    dbot_score = Common.DBotScore(
+        indicator=domain,
+        indicator_type=DBotScoreType.DOMAIN,
+        integration_name="ReversingLabs TitaniumCloud v2",
+        score=0,
+        reliability=RELIABILITY
+    )
+
+    indicator = Common.Domain(
+        domain=domain,
+        dbot_score=dbot_score
+    )
+
+    results = CommandResults(
```

Comment:
the same as https://github.com/demisto/content/pull/30507#discussion_r1376014643

Reply:
Same answer.